composer.json

```diff
@@ -28,7 +32,8 @@
         "php": ">=7.1.0",
         "aws/aws-sdk-php": "^3.0",
         "bbc-rmp/cloudwatch-monitoringhandler": "^1.0",
-        "bbc/programmes-pages-service": "^3.1.0",
+        "bbc/programmes-pages-service": "^4.0",
+        "bbc/programmes-caching-library": "^1.0",
```

Comment:
Do you need to state this here  if it is already a dependency of bbc/programmes-pages-service? I would have thought the version bump to programmes-pages-service would have been sufficient